Form/Type/OptionType.php

```diff
@@ -89,25 +100,16 @@ private function addIntegerType(FormBuilderInterface $builder, $key, OptionInter
      */
     public function configureOptions(OptionsResolver $resolver)
     {
-        $resolver->setDefaults(array(
-            'data_class' => 'Hackzilla\Bundle\PasswordGeneratorBundle\Entity\Options',
-            'csrf_protection' => false,
-        ));
-    }
-
-
-    /**
-     * {@inheritdoc}
-     */
-    public function setDefaultOptions(OptionsResolverInterface $resolver)
-    {
-        $this->configureOptions($resolver);
+        $resolver->setDefaults(
+            array(
```

Comment:
Short array syntax could be used here.

BTW, you should change [`.php_cs`](https://github.com/hackzilla/password-generator-bundle/blob/e740565f6e769bf095b6a06f9c94a280fd651d16/.php_cs#L13) configuration.